gui/elements/qt/_control_bars.py

```diff
@@ -0,0 +1,48 @@
+from PyQt5.QtWidgets import QHBoxLayout, QWidget
+from PyQt5.QtCore import QSize
+
+from ._range_slider import QVRangeSlider
+
+
+class QtControlBars(QWidget):
+    def __init__(self, control_bars):
+        super().__init__()
+
+        self.control_bars = control_bars
+
+        layout = QHBoxLayout()
+
+        # Gamma Slider
+        # self.gammaSlider = QVRangeSlider(slider_range=[-5.0, 5.0, 0.02], values=[-2.5, 2.5])
```

Comment:
Shouldn’t this be a regular `VSlider`?

Reply:
Yes it should, I've just removed this comment for now. Will be very easy to add it back in when needed